### PR TITLE
Add APIM678ApplicationCreationTestCase

### PR DESCRIPTION
**Pull request by Krishanx92**

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java [hunk 1]
```diff
@@ -108,262 +106,146 @@ public void setEnvironment() throws Exception {
...
-        //Error when application name string empty (JIRA 4040)
-        //assertTrue(addApplicationJsonObject.getBoolean("error"), "Error in Application Creation with Invalid data: "
-        // + applicationName);
-
-        //Verify application is exists
-        HttpResponse getAllApplicationsResponse = apiStore.getAllApplications();
-        JSONObject allApplicationsJsonObject = new JSONObject(getAllApplicationsResponse.getData());
-//        assertFalse(allApplicationsJsonObject.getBoolean("error"), "Error in App verification");
-    }
+//    @Test(groups = "webapp", dataProvider = "createApplicationWithInValidData",
```
Comment (isharac) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: Do we need this commented lines?

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
  *
- * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (isharac) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: we do not need to change the year

Comment (Krishanx92) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: Fixed

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java (hunk 1 above)
Comment (Krishanx92) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIM678ApplicationCreationTestCase.java]: Yes. its comment until fix: https://github.com/wso2/product-apim/issues/6012

**Pull request merged**